apis/i18n.d.ts

```diff
@@ -0,0 +1,46 @@
+import { csn, entity } from './cds'
+
+
+declare class I18nBundle {
+  constructor(options: I18nFilesOptions)
+  for(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  at(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  file: string
+  files: I18nFiles
+  get defaults(): Record<string, string>
+  get fallback(): Record<string, string>
+  key4(definition: entity): string
+  texts4 (locale: string): Texts
+  translations4 (...locales : 'all' | string[]) : { [locale]: Texts }
+}
+
+declare class I18nFacade {
```

Comment:
this should be an interface, as we should not insinuate users can do `new I18nFacade()`

Reply:
Valid point! I have changed it